simulator.go

```diff
@@ -0,0 +1,108 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	"github.com/bitrise-io/go-utils/command"
+	"github.com/bitrise-io/go-utils/errorutil"
+	"github.com/bitrise-io/go-utils/log"
+	"github.com/bitrise-io/go-utils/ziputil"
+)
+
+func simulatorBoot(id string) error {
```

Comment:
What is the difference between this simulator boot and `simulator.BootSimulator`?
Should not the content of this file go to that `simulator` package?